pkg/retry/retry_test.go

```diff
@@ -19,39 +19,33 @@ import (
 	"testing"
 	"time"
 
-	"github.com/pingcap/check"
 	"github.com/pingcap/errors"
-	"github.com/pingcap/ticdc/pkg/util/testleak"
+	"github.com/stretchr/testify/require"
 )
 
-func Test(t *testing.T) { check.TestingT(t) }
-
-type runSuite struct{}
-
-var _ = check.Suite(&runSuite{})
-
-func (s *runSuite) TestShouldRetryAtMostSpecifiedTimes(c *check.C) {
-	defer testleak.AfterTest(c)()
+func TestShouldRetryAtMostSpecifiedTimes(t *testing.T) {
+	t.Parallel()
 	var callCount int
 	f := func() error {
 		callCount++
 		return errors.New("test")
 	}
 
 	err := Run(500*time.Millisecond, 3, f)
```

Comment:
i think we could rm the Run func since it is been announced Deprecated for  a while, i will make the change in another diff

Reply:
FYI https://github.com/pingcap/ticdc/pull/2943

Reply:
can github add pr deps? like hold pr2 untill pr1 merge.  Or we must handle this situation case by case.   

Do your suggest to hold this pr?? @ben1009

Reply:
@hi-rustin any suggestion?